jdaviz/configs/specviz/plugins/viewers.py

```diff
@@ -226,21 +227,20 @@ def get_scales(self):
             self.toolbar.active_tool = None
         return {'x': fig.interaction.x_scale, 'y': fig.interaction.y_scale}
 
-    def plot_spectral_line(self, line, scales=None, plot_units=None, **kwargs):
-        if isinstance(line, str):
+    def plot_spectral_line(self, line, plot_units=None, **kwargs):
+        if type(line) == str:
```

Comment:
Why this change?

Reply:
I suspect this was caused by a missed change while rebasing... I'll look into that and revert once I confirm if that was the case.

Reply:
although `scales` was intentionally removed as that logic is now handled by the `viewer/self` reference in the `SpectralLine` object itself.